sips/sip-015/sip-015-network-upgrade.md

```diff
@@ -0,0 +1,1789 @@
+# Preamble
+
+SIP Number: 015
+
+Title: Stacks Upgrade of Proof-of-Transfer and Clarity
+
+Authors:
+    Aaron Blankstein <aaron@hiro.so>,
+    Mike Cohen <mjoecohen@gmail.com>,
+    Greg Coppola <greg@hiro.so>,
+    Brice Dobry <brice@hiro.so>,
+    Hero Gamer <herogamerthesht572@gmail.com>
+    Matthew Little <matthew@blockstack.com>,
+    Jenny Mith <jenny@stacks.org>,
+    Jude Nelson <jude@stacks.org>,
+    Pavitthra Pandurangan <pavitthra@hiro.so>,
+    Rena Shah <rena@trustmachines.co>,
+    Hank Stoever <hank@mechanism.so>,
+    Igor Sylvester <igor@trustmachines.co>,
+    Jesse Wiley <jw@stacks.org>,
+
```

Comment:
I'd like to propose "**Jason**" from CityCoin to be included in Author list. 
From the things I can see and working with him, he has actively been paying attention Stacks 2.1, contributed to any conversation wherever possible on SIP calls and Stacks Forum and discord. He also built the SIP-015.xyz website for the users, contributed to EcosystemDAO voting website, lead the Governance CAB, contributing heavily to the governance lens input regarding SIP-015 activation.
Jason Schrader jason@joinfreehold.com

Reply:
```suggestion
    Jason Schrader <jason@joinfreehold.com>,

```
This should be a one-click commit for anyone with write access

Reply:
Oh, he's already been added, this can be marked as resolved